examples/index.json

```diff
@@ -1467,6 +1467,13 @@
         "verifyBuild": true,
         "verifyOutput": true,
         "isLearnByExample": true
+      },
+      {
+        "name": "Multiple receive",
```

Comment:
The correct order is https://github.com/ballerina-platform/ballerina-dev-website/issues/8856#issuecomment-2044468508
Need to move this next to the `Inter-worker message passing`